Primal/State/Managers/LoginManager.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  LoginManager.swift
+//  Primal
+//
+//  Created by Nikola Lukovic on 18.7.23..
+//
+
+import Foundation
+
+enum LoginState {
+    case notLoggedIn
+    case nsecLoggedIn
+    case npubLoggedIn
```

Comment:
rename into "LoginMethod", with two cases "nsec" and "npub", and represent notLoggedIn with null